cmd/notation/verify.go

```diff
@@ -101,10 +99,12 @@ func runVerify(command *cobra.Command, opts *verifyOpts) error {
 		MaxSignatureAttempts: math.MaxInt64,
 	}
 
-	// core process
+	// core verify process.
 	_, outcomes, err := notation.Verify(ctx, verifier, sigRepo, verifyOpts)
-	if err != nil {
-		logger.Error(err)
+	var errorVerificationFailed *notation.ErrorVerificationFailed
+	if err != nil && !errors.As(err, &errorVerificationFailed) {
+		// print out the error if it's not the general ErrorVerificationFailed
+		fmt.Println("Verification error:", err)
 	}
```

Comment:
Check the type of the error only if there is an error.
```suggestion
	if err != nil {
		var errorVerificationFailed *notation.ErrorVerificationFailed
		if !errors.As(err, &errorVerificationFailed) {
			// print out the error if it's not the general ErrorVerificationFailed
			fmt.Println("Verification error:", err)
		}
	}
```